README.md

```diff
@@ -1,2 +1,5 @@
-# trials-project-review
-Code review project
+# FCC Test Suite Template
```

Comment:
You would like to change the documentation and add a formal and professional description.

Reply:
Good job with the review!. However, we want our comments to be clear and easy to understand. For Example:

This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

This is not a clear comment: You would like to change the documentation and add a formal and professional description.